dataland-backend/src/main/kotlin/org/dataland/datalandbackend/utils/StorageClientUtils.kt

```diff
@@ -0,0 +1,39 @@
+package org.dataland.datalandbackend.utils
+
+import org.dataland.datalandbackend.services.StreamingExternalStorageControllerApi
+import org.dataland.datalandexternalstorage.openApiClient.api.ExternalStorageControllerApi
+import org.springframework.beans.factory.annotation.Autowired
+import org.springframework.stereotype.Component
+import java.io.InputStream
+
+/**
+ * The class holds the various storage clients for the data manager services
+ */
+
+@Component("StorageClientUtils")
+class StorageClientUtils(
+    @Autowired private val streamingStorageClient: StreamingExternalStorageControllerApi,
+    @Autowired private val storageClient: ExternalStorageControllerApi,
+) {
+
+    /**
+     * his methods retrieves an document by using the external storage client and returns the document as an inputstream
+     * @param eurodatId the documentId for the document to be retrieved
+     * @param correlationId the correlationId of the request which caused the exception to be thrown
+     */
+    fun getBlobFromExternalStorageClient(eurodatId: String, correlationId: String): InputStream {
```

Comment:
Würde das einfach "getBlobFromExternalStorage" nennen. Du holst es ja nicht vom client, sondern via den client.

Reply:
Typo in doc:  Startet mit "his nethod"
Text zu "eurodatId" ist falsch: Dort steht noch "documentId"

Reply:
Geändert